bundles/org.openhab.binding.airq/src/main/java/org/openhab/binding/airq/internal/airqHandler.java

```diff
@@ -0,0 +1,956 @@
+                        }
+                        newobj.addProperty("reset", wifidataobj.get("reset").getAsString());
+                        logger.info("air-Q - airqHandler - handleCommand(): dummy to change settings: {}",
+                                newobj.toString());
+                        changeSettings(newobj);
+                    } else {
+                        logger.error("Cannot extract wlan data from this string: {}", wifidatael);
+                    }
+                    break;
+                case "TimeServer":
+                    newobj.addProperty(channelUID.getId(), command.toString());
+                    changeSettings(newobj);
+                    break;
+                case "nightmode_StartDay":
+                    if (isTimeFormat(command.toString())) {
+                        subjson.addProperty("StartDay", command.toString());
+                        newobj.add("NightMode", subjson);
+                        changeSettings(newobj);
+                    } else {
+                        logger.error(
+                                "air-Q - airqHandler - handleCommand(): {} should be set to {} but it isn't a correct time format (eg. 08:00)",
+                                channelUID.getId(), command.toString());
+                    }
+                    break;
+                case "nightmode_StartNight":
+                    if (isTimeFormat(command.toString())) {
+                        subjson.addProperty("StartNight", command.toString());
+                        newobj.add("NightMode", subjson);
+                        changeSettings(newobj);
+                    } else {
+                        logger.error(
+                                "air-Q - airqHandler - handleCommand(): {} should be set to {} but it isn't a correct time format (eg. 08:00)",
+                                channelUID.getId(), command.toString());
+                    }
+                    break;
+                // TODO
+                case "geopos":
+                    break;
+                case "nightmode_BrightnessDay":
+                    break;
+                case "nightmode_BrightnessNight":
+                    break;
+                case "RoomType":
+                    break;
+                case "Logging":
+                    break;
+                case "SecondsMeasurementDelay":
+                    break;
+                case "Rejection":
+                    break;
+                default:
+                    logger.error(
+                            "air-Q - airqHandler - handleCommand(): unknown command {} received (channelUID={}, value={})",
+                            command, channelUID, command);
+            }
+        } else if (command instanceof RefreshType) {
+            if (pollingJob != null) {
+                // pollingJob.notify();
+                // TODO: handle data refresh
+            }
+        }
```

Comment:
Please remove all code that is commented out. Currently all these lines of code do nothing.

Reply:
Concerning all commented code: some is not necessary and can be deleted. However, there are parts (at the end of the file) that are there because I am developing additional functions, but don't want to have them included in this version of the code because they don't work yet (getting historic files). How can I handle those code snippets?

Reply:
I will leave that to the next person that does a review, as I do not know the answer.